src/cmd/create.go

```diff
@@ -710,16 +710,33 @@ func pullImage(image, release, authFile string) (bool, error) {
 		shouldPullImage = true
 	}
 
+	imageSizeChan := make(chan string)
+
 	if promptForDownload {
+
 		fmt.Println("Image required to create toolbox container.")
+		// Launch a goroutine to call getImageSizeFromRegistry and send the size to the channel
+		go func() {
+			imageSize, err := getImageSizeFromRegistry(imageFull)
+			if err != nil {
+				logrus.Debugf("Getting image size failed: %s", err)
+				imageSize = ""
+			}
+			imageSizeChan <- imageSize
+		}()
 
-		var prompt string
+		// Show the download question immediately
+		prompt := fmt.Sprintf("Download %s (...)? [y/N]:", imageFull)
+		fmt.Println(prompt)
 
-		if imageSize, err := getImageSizeFromRegistry(imageFull); err != nil {
-			logrus.Debugf("Getting image size failed: %s", err)
-			prompt = fmt.Sprintf("Download %s? [y/N]:", imageFull)
-		} else {
+		// Wait until the size is received
+		imageSize := <-imageSizeChan
+
+		// Update the string with the image size if it was received or skip the size if it was not obtained
+		if imageSize != "" {
 			prompt = fmt.Sprintf("Download %s (%s)? [y/N]:", imageFull, imageSize)
+		} else {
+			prompt = fmt.Sprintf("Download %s? [y/N]:", imageFull)
```

Comment:
This doesn't replace the prompt in-place, but instead adds a new prompt on the next line, which is odd.